client/payment-request/blocks/payment-request-express.js

```diff
@@ -7,9 +7,7 @@ import { Elements, PaymentRequestButtonElement } from '@stripe/react-stripe-js';
  * Internal dependencies
  */
 import { useInitialization } from './use-initialization';
-import { GooglePayButton } from './branded-buttons';
```

Comment:
The `GooglePayButton` component and its styles (`.gpay-button`) don't seem to be used anywhere else - can you please remove them?